localisation/Idea_Variation_ideas_l_german.yml

```diff
@@ -143,22 +143,24 @@
  monarchie7_desc: "Falls es zum Krieg kommt stellt unser Adel örtliche Millizen auf, die bei der Verteidigung des Landes helfen. \nErmöglicht §GVerpflichtet die lokale Milliz§W Stand Interaktion"
  monarchie5: "Erhöhung der Kronautorität"
  monarchie5_desc: "Durch Erhöhung unserer Autorität ist es uns einfacher die Kontrolle im Land zu behalten und dessen Stabilität zu sichern."
+ 
  republik0: "Republikanische Ideen"
  republik0_desc: "Die republikanische Regierungsform wird von unserem Staat verkörpert."
- republik7: "Römische Vorväter"
- republik7_desc: "Wir können viel von den Römern lernen, die vor uns kamen. Sie schafften es in Zeiten der Republik ein großes Reich aufzubauen und es relativ zentral zu verwalten."
+ republik1: "Diplomatische Mission"
+ republik1_desc: "Wir müssen uns als Republik stärker diplomatisch absichern als manche Monarchien. Denn unsere Art zu Leben stellt Chance und Gefahr zu gleich da."
```

Comment:
We are generally no longer updating German localization, only the default English. If this is a lot of extra effort for you, you can safely skip this next time.